app/views/shared/_error_messages.html.erb

```diff
@@ -0,0 +1,13 @@
+<% if @user.errors.any? %>
+  <div id="error_explanation">
+    <div class="alert alert-danger">
+      <%= t ".p1" %>
+      <%= pluralize(@user.errors.count, t(".p2")) %>.
```

Comment:
bạn tìm hiểu size, count và length, so sánh sự khác nhau rồi trả lời vào đây
Chọn 1 trong 3 cái đó để sử dụng trong trường hợp này và giải thích tại sao nữa nha

Reply:
count truy vấn ra số lượng activerecord sau đó đếm số lượng ,count còn có thể sử dụng kèm điều kiện để tính toán số phần tử, length sẽ đếm số lượng nhưng không truy vấn vào đối tượng đó nếu đối tượng đã được nạp nhưng từ lần thứ 2 trở đi sẽ không load lai mà trả về ngày kết quả , size giống với length nhưng nó thực hiện lên chính đối tượng đó trong bộ nhớ vì thế trong trường hợp này nên dùng size là tốt nhất